kili/project/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Project module."""
+
+
+class Project:  # pylint: disable=too-few-public-methods
```

Comment:
I don't think you should directly have a folder "project" at the root, but it depends on what you want to do. Do we intend to have a folder dataset, asset, label, etc... ? If so then we should have a folder above, named `entities` or something along those lines

Reply:
That is what has been seen with Pierre and Edouard. It comes into the API v3 that we would like to deliver in the next quarter with a better structure, better UX and better function names. one idea would be to have several modules, one with project for example, to work on a specific project, one for automl later, one for managing its organization etc. This creates better scaling of the SDK and allows to work more easily with projects.
A more precise spec will come later, after the CLI
```
project = kili.get_project('<project_id>')
project.import(files)
```